src/components/PersonRow/PersonRow.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type PersonProps = {
+  person: Person;
+  isSelected: boolean;
+};
+
+export const PersonRow: React.FC<PersonProps> = ({ person, isSelected }) => {
+  const {
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+    mother,
+    father,
+  } = person;
+
+  return (
+    <tr
+      data-cy="person"
+      key={slug}
```

Comment:
The 'key' attribute should not be used on a 'tr' element. It should be applied to the parent component when mapping over an array of data. The 'key' attribute can be removed in this case. 🔑